trigger/host.go

```diff
@@ -148,7 +148,7 @@ func (t *hostTriggers) generateTemporaryExemptionExpiration(sub *event.Subscript
 	var err error
 	switch sub.Subscriber.Type {
 	case event.EmailSubscriberType:
-		payload, err = t.templateData.hostEmailPayload(expiringHostTemporaryExemptionEmailSubject, expiringHostEmailBody, t.Attributes())
+		payload, err = t.templateData.hostEmailPayload(expiringHostTemporaryExemptionEmailSubject, expiringHostTemporaryExemptionEmailBody, t.Attributes())
```

Comment:
I'm surprised our linter doesn't catch that `expiringHostTemporaryExemptionEmailBody` wasn't used before. I feel like this type of error happens quite a bit, should we make a ticket to make a test to either improve the linter to catch these or make a new test that catches it? (I can ticket if you think so!)

Reply:
Yeah good call, I'll add a comment to [DEVPROD-7742](https://jira.mongodb.org/browse/DEVPROD-7742) because it seems related